.travis.yml

```diff
@@ -0,0 +1,41 @@
+dist: trusty
+sudo: required
+
+language: c
+# git submodules workaround
+git:
+  submodules: false
+# use sed to replace SSH URL with the public URL, then init submodules 
+before_install:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+  - git submodule update --init --recursive
```

Comment:
libavtp doesn't have any submodule so is this really required? Also, please review this whole file it looks we have more commands not required.

Reply:
Done. Reviewed packages in .travis.yml and updated.